plugins/item_previews/plugin.cmake

```diff
@@ -0,0 +1 @@
+add_web_client_test(item_previews "${PROJECT_SOURCE_DIR}/plugins/item_previews/plugin_tests/item_previews.js" PLUGIN item_previews)
```

Comment:
Let's also add an eslint test to this to make sure it conforms to our style rules and gets static analysis.